__tests__/backend/userCourseController.tests.mjs

```diff
@@ -0,0 +1,90 @@
+import admin from '../../server/config/firebase.mjs';
+import { enrollInCourse, getUserCourses } from '../../server/controllers/userCourseController.mjs';
+import supertest from 'supertest';
+import express from 'express';
+
+const app = express();
+app.use(express.json());
+
+// Middleware to simulate authenticated user
+const simulateAuth = (req, res, next) => {
+  req.user = { uid: 'testUserId' }; 
+  next();
+};
+
+// Route setup for user course endpoints
+app.post('/user/enroll', simulateAuth, enrollInCourse);
+app.get('/user/:userId/courses', simulateAuth, getUserCourses);
+
```

Comment:
There are more endpoints in the userCourseController.mjs file, such as getSuggestedCoursesForUser and getCoursesForUser. You might need to add more test cases for these.

Reply:
Same, these new endpoints were not in the main when I was doing the tests. I will address this in the next PR as soon as all the new changes in the main are merged.